### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by emiel**
I've been spending quite a bit of time with the book and decided to try and contribute a little as I learn more about PureScript.

Here is some preliminary work you may pull in if you like. I am curious what the plans are to make an updated version of the book available. I noticed that Phil Freeman's version of the purescript-book repository have been archived and that there are many forks. Has Phil contributed the book to the community and if so what is the licensing?

### CODE REVIEW

**Comment by dwhitney** [comment]
Hey thanks! The jump from 0.11 to 0.12 was so huge that I decided the book needed a rewrite, and at that time I was on a month long contract that had me commuting via train about 3 hours/day, so I had some time to do the updates, but now I don't, so I don't really have any plans beyond merging pull requests. As for the license - no idea. Maybe creative commons? I had a couple of emails with Phil before I started the project, and he gave me the go ahead.

The right way forward is probably to start a topic on the PureScript Discourse forum and get a community effort going to update the book in total. Then merge it into github.com/purescript-contrib or something... I'll go post!

**Comment by dwhitney** [comment]
https://discourse.purescript.org/t/the-future-of-the-purescript-book/969

File: text/chapter2.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ To write the `diagonal` function (the goal of this chapter), we will need to be
...
+The `purescript-math` library sources should now be available in the `.psc-package/psc-0.12.0/math` subdirectory, and will be included when you compile your project. Note that we did not include `purescript-` from the full package name `purescript-math`. By default, Pulp uses [Bower](https://bower.io/) for dependency management, and for some nuanced reason Bower necessitated the prefix `purescript-` in front of PureScript library names. The PureScript community is slowly moving away from Bower in favor of `psc-package` and the `purescript-` prefix has been dropped. If you forget to drop the prefix, `psc-package` will give you a helpful reminder.
```
[the remark this answers is not in the document]
Comment (emiel) [reply text/chapter2.md]: A later commit will need to fix that yes. Here I was focusing on grammar. ;)

The fact that paragraphs are on a single line makes diffs hard to read. It appears GitHub gives up showing where the differences are when a certain line length is exceeded.

[the remark this answers is not in the document]
Comment (dwhitney) [reply text/chapter2.md]: Good point

**Pull request merged**